Update ACs in 0007 to cover PnL test (#1306)
vegaprotocol/specs: protocol/0007-POSN-non_core_positions_api.md

## Acceptance Criteria

> The Position API stores a net position for each trader who has ever traded in a market. Specifically,

Reviewer: ```suggestion
The Position API provides access to net position per party per market. Specifically,
```

* Buyer P&L (0 if buyer is not closing volume)
* Seller P&L (0 if seller is not closing volume)

## Position and PnL test
- [ ] when a party has never had a position, the realised PnL and unrealised PnL should be 0 (<a name="0007-POSN-009" href="#0007-POSN-009">0007-POSN-009</a>)

> - [ ] when a party has a non-zero postion, and no closeout position, only the unrealised PnL should be changed by change of position or change of mark price while realised PnL should not be updated (<a name="0007-POSN-010" href="#0007-POSN-010">0007-POSN-010</a>)

Reviewer: ```suggestion
- [ ] when a party has a non-zero postion, and has not closed any part of it, only the unrealised PnL should be changed by increase in position or change of mark price while realised PnL should stay constant (<a name="0007-POSN-010" href="#0007-POSN-010">0007-POSN-010</a>)
```

* Buyer P&L (0 if buyer is not closing volume)
* Seller P&L (0 if seller is not closing volume)

## Position and PnL test

> - [ ] when a party does not have any postion, the realised PnL and unrealised PnL should be 0 (<a name="0007-POSN-009" href="#0007-POSN-009">0007-POSN-009</a>)

Reviewer 2: Should this be "has never had a position"? Because a trader with a 0 position *now* could have had one in the past and so have non-0 realised PnL

> - [ ] when a party has a non-zero postion, and no closeout position, the unrealised PnL should be only impacted by change of position or change of mark price while realised PnL should not be updated (<a name="0007-POSN-010" href="#0007-POSN-010">0007-POSN-010</a>)

Reviewer 2: Should this be "Only the unrealised PnL should be change by.." rather than "the unrealised PnL should be only changed by..." ?

- [ ] when a party has a position which gets (partially) closed-out, the unrealised PnL should reflect the change of position while the realised PnL reflect the closed-out position (<a name="0007-POSN-011" href="#0007-POSN-011">0007-POSN-011</a>)

> - [ ] During settlement, unrealised PnL and realised PnL should be unchanged (<a name="0007-POSN-012" href="#0007-POSN-012">0007-POSN-012</a>)

Reviewer 2: Surely unrealised should become 0 and realised should update?
